test/api/kuzzle.test.js

```diff
@@ -37,6 +40,41 @@ describe('/lib/api/kuzzle.js', () => {
     kuzzle.emit('event');
   });
 
+  describe('#throw', () => {
+    it('should throw an ExternalServiceError with right name, msg and code', () => {
+      try {
+        kuzzle.throw('api', 'server', 'elasticsearch_down', '{"status":"red"}');
```

Comment:
(nitpicking) should.js has an API for that (https://shouldjs.github.io/#assertion-throw):

```js
      should(() => kuzzle.throw('api', 'server', 'elasticsearch_down', '{"status":"red"}'))
        .throw(
          ExternalServiceError,
          {
            errorName: 'api-server-elasticsearch_down',
            code: 1,
            message: 'ElasticSearch is down : {"status":"red"}'
          });
```